src/IdentityServer4.RavenDB.Storage/Helpers/DocumentStoreHelper.cs

```diff
@@ -23,14 +23,6 @@ public static DocumentStore InitializeDocumentStore(Action<DocumentStore> config
                 throw new InvalidOperationException($"{nameof(documentStore.Urls)} cannot be empty when setting up Identity Server configuration and operational RavenDb stores.");
             }
 
-            if (documentStore.Certificate != null)
-            {
-                documentStore.AfterDispose += (sender, args) =>

```

Comment:
Why remove the dispose here?

Reply:
I believe it is a user that should be responsible for defining the handler's logic and be aware that a certificate needs to be displosed of along with the displosal of the document store. Do you believe we the library should perform the check and displose of the resource quitely?

Reply:
The problem is that if this isn't disposed, there is a temp file that isn't being cleaned.
This is probably disposed because otherwise users tend to not pat attention to that.

Reply:
👍 reverted